person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(id, age, parent_permission: true, name: 'unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- [ ] As per requirements,kindly consider taking this method under `private` as it should be a private method.
requirement:
 - Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.